linchpin/provision/roles/aws/tasks/teardown_resource_group.yml

```diff
@@ -60,13 +60,26 @@
     topo_output_resources: "{{ topo_output_rundb.output[0]['resources'] }}"
   when: not generate_resources
 
+- name: "print data"
```

Comment:
Do we need this debug statement if not we can remove this can we ? @14rcole

Reply:
Nope, I'll remove it